lib/getPaths.js

```diff
@@ -4,21 +4,39 @@
 */
 "use strict";
 
-module.exports = function getPaths(path) {
+const path = require("path");
+
+const isSubDir = function(parent, dir) {
+	const relative = path.relative(parent, dir);
+	return (
+		relative === "" ||
+		(relative && !relative.startsWith("..") && !path.isAbsolute(relative))
+	);
+};
+
+module.exports = function getPaths(path, jail) {
```

Comment:
The SymlinkPlugin also uses getPaths and need to respect the jail option for symlinked values.

Reply:
Is that really needed if I just care about a jail for the module import? Doesn't the SymlinkPlugin also apply to non module paths? And how could I ensure the jail only applies to symlinked modules?